sale_medical_prescription/tests/test_medical_pharmacy.py

```diff
@@ -16,6 +16,7 @@ def setUp(self):
 
     def test_compute_verified_by_id(self):
         """ Test verified_by_id properly set to user """
+        self.pharmacy_4.is_verified = True
```

Comment:
Again, either the test is bad and more needs to be changed, or it is fine the way it was. Please decide and either revert or fix.